src/thematic/4_vegetation/sfp-401_vegopname-terrest_nl/07_stappenplan.Rmd

```diff
@@ -78,8 +78,8 @@ D. Grootte en vorm van het proefvlak
 De vorm en grootte van het proefvlak is afhankelijk van het vegetatietype en/of
 het project waarin de opnamen kaderen. We raden nadrukkelijk aan om de
 standaardwaarden te gebruiken (zie bijlage 1) en hier enkel van af te wijken om
-gemotiveerde redenen. Voor niet-bosvegetatie is binnen INBO de standaard een 3 m
-x 3 m proefvlak. Voor bossen gaat het om 16 m x 16 m. Voor de onderbouwing van
+gemotiveerde redenen. Voor niet-bosvegetatie is binnen INBO de standaard `r params$length_OV_m`m x 
+`r params$width_OV_m`m proefvlak. Voor bossen gaat het om `r params$length_B_m`m x `r params$width_B_m`m. Voor de onderbouwing van
 deze keuze van proefvlakgrootte zie @RN159, § 4.1.3.3. Er kan
 afgeweken worden van zowel grootte als vorm met een duidelijke motivatie van de
```

Comment:
ook vorm parametrizeren?

Reply:
Just passing by :wink: with a general remark: ``` `r params$length_OV_m`m x ``` should add a space - ``` `r params$length_OV_m` m x ``` - if you want to get `3 m` instead of `3m`